orion/pkg/domains/location.go

```diff
@@ -0,0 +1,65 @@
+package domains
+
+import (
+	"database/sql"
+	"errors"
+	"regexp"
+	"time"
+)
+
+var TABLE_LOCATIONS = "locations"
+
+type Location struct {
+	Id        uint
+	CreatedAt time.Time    `db:"created_at"`
+	UpdatedAt time.Time    `db:"update_at"`
+	DeletedAt sql.NullTime `db:"deleted_at"`
+	LocId     string       `db:"loc_id"	json:"locId"`
```

Comment:
I think CircleCI is complaining about this because it is separated by a tab and not spaces